passmark/passmark_run

```diff
@@ -128,6 +128,7 @@ produce_report()
 		echo Ran > test_results_report
 
 		located=0
+		> results.csv
```

Comment:
Please use rm as it is more obvious what we are doing.

Reply:
The line has been updated to `[ -f results.csv ] && rm results.csv`.

Reply:
LGTM